cEP-0005.md

```diff
@@ -0,0 +1,157 @@
+# coala Configuration
```

Comment:
> It doesn't offer anything more than the ini format

Wrong, it actually does. ini is just a key value store, JSON has types! Same with yaml.

The problem with JSON is that it's a lot to type. INI is so super simple. YAML confuses me whenever I use it.